src/App.vue

```diff
@@ -539,22 +536,23 @@ export default class App extends Vue {
 
   /** Opens Auth Web in a new tab with the survey query param. */
   protected launchFilingSurvey (): void {
-    const iaSurveyId = sessionStorage.getItem('IA_SURVEY_ID')
-
     // safety check
```

Comment:
This method should not be called if IA Survey ID was 0 in the first place, but we still don't want to open a new window with an invalid id.